chart/compass/charts/provisioner/values.yaml

```diff
@@ -13,6 +13,8 @@ gardener:
   project: "" # Gardener project connected to SA
   kubeconfigPath: "/gardener/kubeconfig/kubeconfig"
   kubeconfig: "" # Base64 encoded Gardener SA key
+  tenantConfigPath: "/gardener/tenant/config"
+  tenantConfigMapName: ""
```

Comment:
I think in default installation `tenantConfigPath` should be empty as we want `shouldEnableAuditLogs` to evaluate to false. I would also rename those variables to highlight that that those are audit log tenants, as there are other tenants in the system.